NOTICE.txt

```diff
@@ -1006,6 +1006,50 @@ Files covered:
 spyder/plugins/variableexplorer/widgets/dataframeeditor.py
 
 
+-------------------------------------------------------------------------------
+
+
+
+Classes from objbrowser 1.2.1
+-----------------------------
+
+
+Copyright (c) 2016 Pepijn Kenter
+
+
+Author: Pepijn Kenter | <author-email> | <author-website>
+Site/Source: https://github.com/titusjan/objbrowser
+License: MIT (Expat) License | https://opensource.org/licenses/MIT
+
+Modifications made to each class to adapt them to Spyder's needs.
+
+
+objbrowser is distributed under the MIT license.
+
+
+Use of the core functionality of the project changing the non-patched
+version of qtpy used for the actual qtpy project (since it is already a
+dependency of Spyder)
```

Comment:
Please change this to

> Use the core functionality of the project, mainly changing the code to use
the upstream version of qtpy project (since it is already a dependency of
Spyder).